packages/google_maps_flutter/google_maps_flutter/pubspec.yaml

```diff
@@ -2,11 +2,11 @@ name: google_maps_flutter
 description: A Flutter plugin for integrating Google Maps in iOS and Android applications.
 repository: https://github.com/flutter/plugins/tree/master/packages/google_maps_flutter/google_maps_flutter
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+maps%22
-version: 2.0.8
+version: 2.1.0
```

Comment:
I don't think there's a public API change here so a minor version bump should be enough.

Reply:
Requiring a new Flutter stable isn't cause enough for a minor bump? @stuartmorgan what do you think?

Reply:
I don't think what we're doing really fits into the semver scheme, so I think it doesn't especially matter. Since the resolver will prevent someone for whom this won't work from ever getting it in the first place, and for anyone else it's basically a bugfix (it won't get modified on the fly, and for some packages will stop having so many deprecation warnings) I think a bugfix bump is fine.

But I'm also fine with saying "it's not exactly a bugfix either so lets call it a minor bump". 🤷🏻

(If I'm the tiebreaker here, I think I'd lean slightly toward bugfix bump rather than minor bump).

Reply:
Done.